prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml

```diff
@@ -64,6 +64,7 @@ postsubmits:
     labels:
       preset-prow-deployer-service-account: "true"
```

Comment:
Remove this too? (Is this even an extant preset?)

Reply:
Done